src/omegga/player.ts

```diff
@@ -361,32 +361,34 @@ class Player implements OmeggaPlayer {
       { first: 'index', timeoutDelay: 500 }
     );
 
+    if(pawn === "None") return;
+
     return pawn;
   }
 
-  async getPosition(): Promise<[number, number, number]> {
+  async getPosition(): Promise<[number, number, number] | undefined> {
```

Comment:
Same here -- return `null` instead of `undefined` since `null` represents the absence of a value while `undefined` represents the absence of a definition at any point in time